src/Twig/LocaleExtension.php

```diff
@@ -426,4 +439,35 @@ private function getFlagCodes()
             'ZW' => 'Zimbabwe',
         ]);
     }
+
+    public function localedatetime($dateTime, $format = '%B %e, %Y %H:%M', $locale = 0)
```

Comment:
types

Reply:
Added them, except for `$dateTime`, which is either `string` or `\DateTime`